src/Instrumentation/OpenSearchInstrumentation.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Shopware\OpenTelemetry\Instrumentation;
+
+use OpenSearch\Client;
+use OpenSearch\Endpoints\AbstractEndpoint;
+use OpenSearch\Namespaces\IndicesNamespace;
+use OpenTelemetry\API\Instrumentation\CachedInstrumentation;
+use OpenTelemetry\API\Trace\Span;
+use OpenTelemetry\API\Trace\SpanKind;
+use OpenTelemetry\API\Trace\StatusCode;
+use OpenTelemetry\Context\Context;
+use OpenTelemetry\SemConv\TraceAttributes;
+use Throwable;
+
+use function OpenTelemetry\Instrumentation\hook;
+
+class OpenSearchInstrumentation
```

Comment:
can we make opensearch optional? register this instrumentation only when opensearch client class exists and move it to require-dev so phpstan is still happy?